contracts/NFTStorefront.cdc

```diff
@@ -72,7 +75,12 @@ pub contract NFTStorefront {
         storefrontResourceID: UInt64, 
         purchased: Bool,
         nftType: Type,
-        nftID: UInt64
+        nftUUID: UInt64,
+        nftID: UInt64,
+        ftVaultType: Type,
+        price: UFix64,
+        customID: String,
+        commissionAmount: UFix64
```

Comment:
instead of `commisionAmount`, can't we have the `saleCuts` (map of address to amount)?

Reply:
The key difference between a `saleCut` vs `commisionAmount` is that the latter can be claimed by any party (dynamic recipient), such as a markteplace, at time the time of purchase . Whereas a `saleCut` requires the recipient to be known ahead of time, such as a royalty recipient (static recipient)

Reply:
i see